### PR TITLE
doc: FOTA over BLE doc update

### PR DESCRIPTION
**Pull request by annakielar**
FOTA over BLE chapter (ug_nrf52.rst) reordered and rephrased. 
1 substitution removed from shortcuts.txt and ug_nrf91_features.

Signed-off-by: Anna Kielar <anna.kielar@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/ug_nrf52.rst [hunk 1]
```diff
@@ -214,38 +214,35 @@ FOTA upgrades can be used to replace the application.
...
-      * |fota_upgrades_req_mcuboot|
+To perform a FOTA upgrade, complete the following steps:
+   1. Create a binary file that contains the new image.
 
-#. Create a binary file that contains the new image.
       |fota_upgrades_building|
       The :file:`app_update.bin` file is the file that must be downloaded to the device.
 
-#. Download the new image to a device.
+   #. Download the new image to a device.
```
Comment (annwoj) [inline doc/nrf/ug_nrf52.rst:234]:
```suggestion
   #. Download the new image to your device.
```

File: doc/nrf/ug_nrf52.rst [hunk 2]
```diff
@@ -214,38 +214,35 @@ FOTA upgrades can be used to replace the application.
 FOTA over Bluetooth LE
```
Comment (annwoj) [inline doc/nrf/ug_nrf52.rst:214]: Bluetooth LE or Bluetooth Low Energy - to be unified in this section

File: doc/nrf/ug_nrf52.rst [hunk 3]
```diff
@@ -211,41 +211,40 @@ FOTA upgrades can be used to replace the application.
...
    |note|
 
-FOTA over Bluetooth LE
+FOTA over Bluetooth Low Energy
 ======================
 
-To perform a FOTA upgrade, complete the following steps:
+To enable support for FOTA upgrades, do the following:
+
+   * Use :doc:`mcuboot:index-ncs` as the upgradable bootloader (:kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` must be enabled).
```
Comment (FrancescoSer) [inline doc/nrf/ug_nrf52.rst:219]: Consider that :doc:`mcuboot:index-ncs`  is named "Introduction to MCUboot", so the sentence structure should accomodate the link accordingly

File: doc/nrf/ug_nrf52.rst [hunk 4]
```diff
@@ -211,41 +211,40 @@ FOTA upgrades can be used to replace the application.
 .. note::
    |note|
 
-FOTA over Bluetooth LE
+FOTA over Bluetooth Low Energy
 ======================
```
Comment (FrancescoSer) [inline doc/nrf/ug_nrf52.rst:215]:
```suggestion
FOTA over Bluetooth Low Energy
==============================
```

File: doc/nrf/ug_nrf52.rst [hunk 5]
```diff
@@ -211,41 +211,40 @@ FOTA upgrades can be used to replace the application.
...
 
-To perform a FOTA upgrade, complete the following steps:
+To enable support for FOTA upgrades, do the following:
+
+   * Use :doc:`mcuboot:index-ncs` as the upgradable bootloader (:kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` must be enabled).
 
-1. Make sure that your application supports FOTA upgrades.
-      To download and apply FOTA upgrades, the following requirements apply:
+   * Enable the mcumgr module that handles the transport protocol over Bluetooth Low Energy:
```
Comment (FrancescoSer) [inline doc/nrf/ug_nrf52.rst:221]:
```suggestion
   * Enable the mcumgr module that handles the transport protocol over Bluetooth Low Energy as follows:
```
Just to clarify the sense of the following lines

File: doc/nrf/ug_nrf52.rst [hunk 6]
```diff
@@ -211,41 +211,40 @@ FOTA upgrades can be used to replace the application.
...
-      * You must enable the mcumgr module, which handles the transport protocol over Bluetooth Low Energy.
-        To enable this module in your application, complete the following steps:
+   a. Enable :kconfig:option:`CONFIG_MCUMGR_CMD_OS_MGMT`, :kconfig:option:`CONFIG_MCUMGR_CMD_IMG_MGMT`, and :kconfig:option:`CONFIG_MCUMGR_SMP_BT`.
+   #. Call ``os_mgmt_register_group()`` and ``img_mgmt_register_group()`` in your application.
+   #. Call ``smp_bt_register()`` in your application to initialize the mcumgr Bluetooth Low Energy transport.
 
-        a. Enable :kconfig:option:`CONFIG_MCUMGR_CMD_OS_MGMT`, :kconfig:option:`CONFIG_MCUMGR_CMD_IMG_MGMT`, and :kconfig:option:`CONFIG_MCUMGR_SMP_BT`.
-        #. Call ``os_mgmt_register_group()`` and ``img_mgmt_register_group()`` in your application.
-        #. Call ``smp_bt_register()`` in your application to initialize the mcumgr Bluetooth Low Energy transport.
+   After completing these steps, your application should advertise the SMP Service with UUID 8D53DC1D-1DB7-4CD3-868B-8A527460AA84.
```
Comment (FrancescoSer) [inline doc/nrf/ug_nrf52.rst:227]:
```suggestion
   After completing these steps, your application should advertise the SMP Service with UUID ``8D53DC1D-1DB7-4CD3-868B-8A527460AA84``.
```

File: doc/nrf/shortcuts.txt [hunk 7]
```diff
@@ -60,8 +60,6 @@
 .. |fota_upgrades_def| replace:: You can upgrade the firmware of the device over the air, thus without a wired connection.
    Such an upgrade is called a FOTA (firmware over-the-air) upgrade.
 
-.. |fota_upgrades_req_mcuboot| replace:: To upgrade the application, you must use :doc:`mcuboot:index-ncs` as the upgradable bootloader (:kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` must be enabled).
-
 .. |fota_upgrades_building| replace:: To create a binary file for an application upgrade, make sure that :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` is enabled and build the application as usual.
```
Comment (FrancescoSer) [inline doc/nrf/shortcuts.txt:63]:
```suggestion
.. |fota_upgrades_building| replace:: To create a binary file for an application upgrade, build the application with the :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` option enabled.
```

**Review by FrancescoSer — CHANGES_REQUESTED** [review]
Just a few tips

**Review by sigvartmh — APPROVED** [review]

**Review by sigvartmh — APPROVED** [review]
The instructions are correct in how to enable things I guess there is still some things to touch up on but from a technical stand point this is okei.

**Review by annwoj — APPROVED** [review]

**Review by FrancescoSer — APPROVED** [review]

**Pull request merged**